openmeetings-server/src/site/resources/js/site.js

```diff
@@ -27,7 +27,7 @@ $(document).ready(function() {
 	$('.header .bannerRight').parent().append(
 		$('<div class="float-right">')
 			.append($('<a href="https://www.apache.org/events/current-event" class="apachecon-banner bannerRight">')
-				.append($('<img src="https://www.apache.org/events/current-event-234x60.png">'))
```

Comment:
This URL doesn't work anymore. It points to the old website and image.

Reply:
@solomax happy with that? We can fix it once/if current-event image URL starts working again

Reply:
Maybe Im misleading here, but the link target is still the same: https://www.apache.org/events/current-event

I have just updated the image URL, since that one doesn't seem to work anymore.

So whimsy should be still correct and pass:https://whimsy.apache.org/site/check/events